### DISCUSSION TITLE
为什么播放YOUTUBE视频错误

### DISCUSSION THREAD

**Discussion by sssdexppp** [ZuodaoTech/everyone-can-use-english#414] [Q&A]
你好 非常好的软件 但是我直接播放YOUTUBE视频显示无法下载 请问应该怎么解决 谢谢

**Comment by an-lee** [comment]
每个视频都失败吗？会不会是网络问题？应该需要全局代理才能下载成功。

**Reply by ruruqui** [reply]
老师我也是无法下载视频，已经是全局代理了还是这样

**Comment by an-lee** [comment]
Finished in [v0.2.4](https://github.com/xiaolai/everyone-can-use-english/releases/tag/v0.2.4)
